amr-wind/core/vs/tensorI.H

```diff
@@ -153,7 +153,7 @@ operator&&(const TensorT<T>& t1, const TensorT<T>& t2)
 template <typename T>
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE T mag_sqr(const TensorT<T>& t)
 {
-    return (t && t);
+    return (t);
```

Comment:
This is not correct.

Reply:
Interesting. I just looked this up. I didn't know it was a C++11 thing.